public/styles/main.css

```diff
@@ -124,6 +124,57 @@ a.connections-table.icon {
   padding: 1rem;
 }
 
+/* Form related components */
+
+.new-connection{
+  background-color: #fff;
+  overflow-x: auto;
+  display: flex;
+  flex-direction: row;
+  border-radius: 12px;
+  padding: 1rem;
+  max-width: max-content;
+}
+
+.new-connection.col{
+  display: flex;
+  flex-direction: column;
+  max-width: 70ch;
+}
+
+.new-connection input {
+  display: flex;
+  border-color: #abafb1;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 8px;
+  align-self: stretch;
+}
+
+.new-connection button {
+  display: flex;
+  border-color: #abafb1;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 8px;
+  align-self: stretch;
+}
```

Comment:
I guess this can be ignored since it's mention in the PR that this is excluding styling.